pkg/hub/managedclusterset/default_managedclusterset_controller.go

```diff
@@ -19,7 +19,11 @@ import (
 	clusterv1beta1 "open-cluster-management.io/api/cluster/v1beta1"
 )
 
-const defaultManagedClusterSetName = "default"
+const (
+	defaultClusterSetAnnotationKey           = "open-cluster-management.io/defaultclusterset"
```

Comment:
I think we should check what is the current annotation set by bootstrap rbac to make it unmanaged. We should use the similar way. Note we probably have other bootstrapped clusterset in the future

Reply:
> I think we should check what is the current annotation set by bootstrap rbac to make it unmanaged. We should use the similar way. Note we probably have other bootstrapped clusterset in the future

found a annotation: [`DeploymentIgnorePodAnnotation = "deploy.openshift.io/deployer-pod.ignore"`](https://github.com/openshift/library-go/blob/master/pkg/apps/appsutil/const.go#L51), so i define the annotation like this: 
[`"open-cluster-management.io/managedclusterset.default.unmanage"`](https://github.com/open-cluster-management-io/registration/pull/210/files#diff-83468281441062b8e876bcef48f61044f897afce081d9790253027dacd34fc78R23) 
if this annotation exists on default clusterset, we will unmanage clusterset, just focus on the annotation key.

Reply:
> I think we should check what is the current annotation set by bootstrap rbac to make it unmanaged. We should use the similar way. Note we probably have other bootstrapped clusterset in the future

Not sure if I got it correctly.

Reply:
I think we should use the annotation like: `open-cluster-management.io/autoupdate:false` which is same as https://kubernetes.io/docs/reference/access-authn-authz/rbac/#auto-reconciliation

Reply:
Same as the rbac logic here: https://github.com/kubernetes/kubectl/blob/0f88fc6b598b7e883a391a477215afb080ec7733/pkg/cmd/auth/reconcile.go
we may just need to check if this annotation exist `open-cluster-management.io/autoupdate:false`

Reply:
@ldpliu how about this? https://github.com/open-cluster-management-io/registration/pull/210/files#diff-83468281441062b8e876bcef48f61044f897afce081d9790253027dacd34fc78R23-R24

Reply:
yes, the code logic looks ok for me, and give another comment about variable name.